cmd/netgoal/generate.go

```diff
@@ -64,6 +65,7 @@ func init() {
 	generateCmd.Flags().IntVarP(&relaysToGenerate, "relays", "R", -1, "Relays to generate")
 	generateCmd.Flags().IntVarP(&nodeHostsToGenerate, "node-hosts", "N", -1, "Node-hosts to generate, default=nodes")
 	generateCmd.Flags().IntVarP(&nodesToGenerate, "nodes", "n", -1, "Nodes to generate")
+	generateCmd.Flags().IntVarP(&nonPartnodesToGenerate, "non-participating-nodes", "X", 0, "Non participating nodes to generate")
```

Comment:
Consider using -h for nonparticipating node count to align with -N and -n for participating nodes.

Reply:
can't. that's hard wired to '--help'

Reply:
In the makefile in the new scenario, maybe we should just use the long-form of the arguments, e.g. `non-participating-nodes`. I found it really difficult when I first looked at our scenarios to understand what all the arguments in the Makefile were referring to.

And with this new change, maybe we would want to modify all the existing Makefiles in the existing scenarios to make sure they match for record keeping? 

I think the next time we use netgoal, this parameter is required, otherwise, default looks like it's set to 0. If the new flag isn't provided, shouldn't it just match the number of `non-participating-nodes-hosts`?